app/models/user.rb

```diff
@@ -83,9 +83,8 @@ def moderatable
                                  :message => MSG_LOGIN_BAD }
 
   #OPTIMIZE: use the same format validations in user, account and invitation, add TESTS!
-  validates :email, length: { :within => 6..255, :allow_blank => true } #r@a.wk
-  validates :email, format: { :with => RE_EMAIL_OK, :allow_blank => false,
-                              :message => MSG_EMAIL_BAD, :unless => :minimal_signup? }
+  validates :email, length: { :within => 6..255, :allow_blank => false } #r@a.wk
+  validates :email, format: { :with => RE_EMAIL_OK, :allow_blank => false, :message => MSG_EMAIL_BAD }
```

Comment:
This allow_blank false should have a test here:  https://github.com/3scale/porta/blob/2ead029a58701b3ff3e196c5338177f4f25a97f6/test/unit/user_test.rb#L3

something like:
```ruby
test 'email cannot be blank' do
  user = FactoryBot.build(:admin)
  user.email = nil
  refute user.valid?
  assert_matches user.errors[:email].to_s, 'too short'
end
```